invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/components/DocumentMetadataAccordion.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { Accordion, Divider, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DocumentRelations, DocumentInfo } from './../components';
+import { DocumentTableOfContent } from './DocumentTableOfContent';
+import { DocumentConference } from './DocumentConference';
+
+export class DocumentMetadataAccordion extends Component {
```

Comment:
Not sure about the naming, since it appears only in the mobile view, and there is no accordion in general in the metadata.

Maybe `DocumentMetadataMobile` makes more sense.. or something more descriptive.

Reply:
Was this the discussion we had today and so you have already decided to call cmps with *Mobile postfix? BTW, I am fine with this! :)

Reply:
Actually I didn't want to use `mobile` in this case, because it might be reused not only for a mobile view but to present the data in different way somewhere else